internal/cli/inputs_test.go

```diff
@@ -218,14 +226,24 @@ func TestResolveGroupID(t *testing.T) {
 			},
 			expectedGroupID: testGroup.ID,
 		},
+		{
+			description:     "Should select default group when FlagAutoConfirm is set",
```

Comment:
[style] for test descriptions, try to keep things all lower-cased (and without punctuation)

```go
description:     "should select default group when provided and ui is set to auto confirm",
```